pysr/sr.py

```diff
@@ -447,7 +447,7 @@ def pysr(X=None, y=None, weights=None,
     global_extra_sympy_mappings = extra_sympy_mappings
 
     print("Running on", ' '.join(command))
-    process = subprocess.Popen(command, stdout=subprocess.PIPE, bufsize=1)
+    process = subprocess.Popen(command, stdout=subprocess.PIPE, bufsize=1, shell=True)
```

Comment:
on a Windows System for some reason without the shell argument I get: 
FileNotFoundError: [WinError 2] The system cannot find the file specified

Solution from: https://stackoverflow.com/questions/42572582/winerror-2-the-system-cannot-find-the-file-specified-python

Reply:
Note that I have continuous testing on windows which works fine without this argument: https://travis-ci.com/github/MilesCranmer/PySR/jobs/466146940, so I think the FileNotFoundError on your system could be from something else.

This change seems to be causing my PySR to hang. I think this `shell=True` argument is what is breaking it. `shell=True` means you are passing a single string, but `shell=False` means you are passing a list - which PySR does. So we will need to change this back.